yarn.lock

```diff
@@ -2556,6 +2573,17 @@
     "@sentry/utils" "7.88.0"
     https-proxy-agent "^5.0.0"
 
+"@sentry/node@^7.92.0":
```

Comment:
looks like lots of duplicates?

make sure you're doing `npx yarn-deduplicate && yarn install`

Reply:
oops good catch. Just ran `yarn-deduplicate` and pushed.